acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/node/NodeConfiguration.java

```diff
@@ -46,4 +48,8 @@ public Duration getPollingInterval() {
   public String getDownstreamUrl() {
```

Comment:
not from this review - but is it actually "downstream" given this belongs to the node?

Reply:
🤔 Yeah, it's not really downstream from the node, it's downstream from the signer

Reply:
renamed